bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/ReadSessionCreator.java

```diff
@@ -18,6 +18,7 @@
 import static com.google.cloud.bigquery.connector.common.BigQueryErrorCode.UNSUPPORTED;
 import static java.lang.String.format;
 
+import com.google.cloud.bigquery.StandardTableDefinition;
```

Comment:
Is it needed?

Reply:
Yes the build is failing https://pantheon.corp.google.com/cloud-build/builds;region=global/bae5775e-f3b0-4967-bd2a-5ffcc459fcfb?mods=dataproc_env_test&project=cloud-dataproc-ci. 
Reason being your changes in PR #674 need `StandardTableDefinition`, but in the PR #673 removed it. Since PR #673 got merged later this issue got created.